test/boost/sstable_compaction_test.cc

```diff
@@ -5017,3 +5017,44 @@ SEASTAR_TEST_CASE(compaction_manager_stop_and_drain_race_test) {
     testlog.info("stopping compaction manager");
     co_await cm.stop();
 }
+
+SEASTAR_TEST_CASE(move_sstables_from_staging_test) {
+    return test_env::do_with_async([](test_env &env) {
+        auto builder = schema_builder("tests", "table_move_sstables_from_staging_test")
+                .with_column("id", utf8_type, column_kind::partition_key)
+                .with_column("value", int32_type);
+        // Such that no compaction is triggered when adding sstables to the table.
+        builder.set_compaction_strategy(sstables::compaction_strategy_type::null);
+        auto s = builder.build();
+
+        auto tmp = tmpdir();
+        auto staging_path = fs::path(tmp.path().string()) / fs::path(sstables::staging_dir);
```

Comment:
`fs::path(tmp.path().string()) == tmp.path()`.
This should boil down to the following simple statement:
```
        auto staging_path = tmp.path() / sstables::staging_dir;
```

Reply:
oops, you're right. will fix.